### ISSUE TITLE
Uwazi restore failing

### ISSUE THREAD

**Issue by tlacoyodefrijol** [huridocs/uwazi#3610]
When following the process to restore the Uwazi database in a new instance, there are 52 documents that fail to restore. These are most likely linked to the Default template, since the template is then empty and all its connections are lost but the cases are there. 

<img width="1380" alt="Screenshot 2021-04-19 at 10 10 06" src="https://user-images.githubusercontent.com/5587545/115204974-5fd49400-a0f9-11eb-9bd9-27bf10c5590a.png">

<img width="1441" alt="Screenshot 2021-04-19 at 10 43 23" src="https://user-images.githubusercontent.com/5587545/115207650-133e8800-a0fc-11eb-9d65-8c8ccb0d7162.png">

**Comment by RafaPolit** [comment]
I am sorry I don't think I understand what is the problem.  Can you describe more what this screen captures are showing?

**Comment by txau** [comment]
@tlacoyodefrijol it looks like a Mongo error. A wild guess: check the version of the database you used for creating the dump and verify if it is the same as the one you are using for the restore.

> These are most likely linked to the Default template, since the template is then empty and all its connections are lost but the cases are there.

I don't understand how you get to that conclusion? At this stage only Mongo is involved, Uwazi or Elastic should not interfere. Also since Mongo is a document DB, we don't even have any foreign key restrictions etc that could prevent the information from being restored.

**Comment by tlacoyodefrijol** [comment]
Hi folks. It was indeed a Mongo error. Basically what happened was: 

- We did a fresh install but ran `yarn blank-state` which created the Index. That's why we saw Document there instead of our template. 
- Since `mongorestore` doesn't rewrite content, everything would be restored except that that was already created by `blank-state`. 
- We did a mongo database drop, then run the restore and build the index again. 

It might be useful to include this nuance for people who might be new to mongo and are running a self-hosted instance.
